### DISCUSSION TITLE
have the opportunity to change the color the text in the checkbox setting

### DISCUSSION THREAD

**Discussion by oswaldB** [Budibase/budibase#2218] [Ideas]
![image](https://user-images.githubusercontent.com/3023526/127875322-6d674b1a-f652-48ed-852b-68295c1ed942.png)

**Comment by mjashanks** [comment]
Try `Edit Custom CSS` and input `--spectrum-fieldlabel-m-text-color: red;`

I found this by inspecting the label, and determining what CSS var is used to colour it:
![image](https://user-images.githubusercontent.com/3524181/127876977-c9d7b2fd-c413-4b91-b3e1-cc902d575cd8.png)

Note - we hope to add global theming soon, which will solve this in a nice way.

**Reply by oswaldB** [reply]
Thank you.

For people that doesn't like css, you can use the container element and put the checkbox and the paragraph element.
